packages/typespec-ts/src/modular/helpers/operationHelpers.ts

```diff
@@ -1350,7 +1357,10 @@ export function serializeRequestValue(
     case "model": // this is to build serialization logic for spread model types
       return `{${getRequestModelMapping(context, type, "").join(",")}}`;
     case "constant":
-      return `${nullOrUndefinedPrefix}${getConstantValue(type)}`;
+      if (isTopLevel) {
```

Comment:
not quite understand this change, why top-level is different that other levels?

Reply:
because for top-level we skip these params in signiture so we would fallback to constant, but other levels we could get that property directly.